src/main/readout.go

```diff
@@ -33,7 +33,7 @@ import (
 	"math"
 	"os"
 	"path/filepath"
-	"github.com/nDenerserve/SmartPi/src/smartpi"
```

Comment:
https://github.com/SuperQ/SmartPi/commit/e1bca4e25a4cfe458efd9c0c4a09cba376735c66

The only annoying thing is you'll need to symlink your fork into `${GOPATH}/src/github.com/nDenerserve/`. for all the code to work on your branch.